workload-elsa-data-aws/elsa-data-stack-settings.ts

```diff
@@ -15,6 +15,11 @@ export interface ElsaDataStackSettings {
    */
   isDevelopment?: boolean;
 
+  /**
+   * Forces a new deployment of all stacks by updating the description. Defaults to false.
+   */
+  forceDeployment?: boolean;
+
```

Comment:
Is there any other way?? A tag? And surely this just forces the stack description to be updated - nothing else will change?

Reply:
I was having some issues with the cdk `--force` flag not working as I expected, so I tried this. Although, I'm not using it anymore, so happy to remove it?